checkov/common/output/common.py

```diff
@@ -15,3 +15,32 @@ class ImageDetails(SCADetails):
     image_id: str = ''
     name: str | None = ''
     related_resource_id: str | None = ''
+
+
+def is_raw_formatted(licenses: str) -> bool:
+    return licenses.__contains__('","')
```

Comment:
```suggestion
    return '","' in licenses
```
you typically don't use the internal methods directly.